src/main/java/Todo.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Encapsulates a todo task which has a description.
+ */
+public class Todo extends Task {
```

Comment:
As with Todo as well, perhaps you could consider camel casing it to ToDo instead. Once again, just a suggestion!